python/pyspark/ml/wrapper.py

```diff
@@ -68,7 +68,7 @@ def _make_java_param_pair(self, param, value):
         """
         sc = SparkContext._active_spark_context
         param = self._resolveParam(param)
-        java_param = self._java_obj.getParam(param.name)
+        java_param = self._java_obj.getParam(param.name.rstrip("_"))
```

Comment:
Since spark.ml NaiveBayes in Scala was not in 1.4, let's rename lambda.  How about we just call it "smoothing" (or another name if you have a strong preference)?